spec/requests/shots_spec.rb

```diff
@@ -2,19 +2,19 @@
 
 describe "Api::V1::Shots" do
   context "POST /api/v1/games/:id/shots" do
-    let(:player_1_board)   { Board.new(4) }
-    let(:player_2_board)   { Board.new(4) }
+    let(:player_1)   { Player.new(Board.new(4),'kasjdfhlai4oeiufrwh')}
```

Comment:
These look like they are different lengths... I'm surprised it just lets you dump in any text string... perhaps an indication that more is needed for APIkey validation from me.